src/Players.js

```diff
@@ -0,0 +1,43 @@
+import { useState, useEffect } from "react"
+
+export default function Envelope() {
+    const defaultPlayerList = ["Avishek Khan", "Brandon Mannarino", "Jeremy Morgan", "Lee Evans", "Donovan Bendana", "Jake Hoberg", "Joshua France", "Miles Todzo", "Luis Antillon"];
+    const [playerList, setPlayerList] = useState(defaultPlayerList);
+
+
+    function People() {
+
+        const [active, setActive] = useState(playerList[0]);
+        const [missionsIndexArray, setMissionsIndexArray] = useState([0, 0, 0, 0, 0, 0]);
+
+        return (
+            <>
+                {playerList.map((player) => (
+                    <button
+                        key={player}
+                        active={active === player}
+                        missionsIndexArray={active === player}
+                        onClick={() => setActive(player)}
+                    >
+                        {player}
+                    </button>
+                ))}
+                <p />
+                <p> Current Selection:  {active} {missionsIndexArray} </p>
+
+            </>
+        );
+    }
+
+
+    return (
+        <>
+            <button
+                onClick={() => setPlayerList(defaultPlayerList)}>Use Sample Player List</button>
+            <button onClick={() => setPlayerList(["test"])}> Use My Own Player</button>
+            
```

Comment:
I need to make a new state control that will allow you to switch between these playerLists. 

That might not even be necessary actually. What I think I should do is set this as the default PlayerList and then you can add and remove those players at the beginnig when you set who's playing this game. 


Maybe an idea: 
Game will begin now boolean: init false 
Display the default player list
Hit confirm () => set GameBeginNow(true)
also calls another function that will create everyone's random arrays

for testing, I will need to display those random arrays, next feature after that is displaying it based on person